### PR TITLE
style: add laravel pint for psr12 code style

### LINKED ISSUE #97: Add a coding style, with an automatic formatter
I think you should add a code formatter to the project, like [PHP-CS-Fixer](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer) or [Laravel Pint](https://github.com/laravel/pint) (build on top of PHP-CS-Fixer).

Would make it easier to contribute without having to worry about code style.

### PR DESCRIPTION
**Pull request by adevade**
## Description

This PR adds [Laravel Pint](https://github.com/laravel/pint) ([docs](https://laravel.com/docs/9.x/pint)) as a dev dependency and runs the formatter with [the PSR12 preset](https://www.php-fig.org/psr/psr-12/).

Pint is build on top of [PHP CS Fixer](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer), and takes care of configuration behind the scenes, making it easy to install, run and maintain.

## Checklist

- [x] Read the [contributing guidelines](CONTRIBUTING.md).
- [x] Each commit follows the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) spec format.

Closes #97

### CODE REVIEW

**Comment by adevade** [comment]
If you'd rather have PHP-CS-Fixer than Laravel Pint, let med know and I'll send a new PR using that. 👍

I haven't used PHP-CS-Fixer personally, but have been using Laravel Pint because it's so easy to setup.

File: src/UrlBuilder.php [hunk 1]
```diff
@@ -2,34 +2,40 @@
...
 
     // define class constants
     // should be private; but visibility modifiers are not supported php version <7.1
-    const TARGET_RATIOS = array(1, 2, 3, 4, 5);
+    public const TARGET_RATIOS = [1, 2, 3, 4, 5];
+
+    public const DPR_QUALITIES = [1 => 75, 2 => 50, 3 => 35, 4 => 23, 5 => 20];
 
-    const DPR_QUALITIES = array(1 => 75, 2 => 50, 3 => 35, 4 => 23, 5 => 20);
+    public const MIN_WIDTH = 100;
```
Comment (luqven) [inline src/UrlBuilder.php:29]: AFAIK, adding `public` to these "private" class variables doesn't change much. So as long as that's the case and it's needed to make the linter happy, I don't have a problem with it.

**Review by luqven — APPROVED** [review]
LGTM 👍🏼 

Thanks for your hard work on this! We might need a rebase here after merging in the unit test PR. I'll ping you when/if it's needed.

File: src/UrlBuilder.php (hunk 1 above)
Comment (adevade) [reply src/UrlBuilder.php:29]: Yep! If not defined, `public` is assumed by PHP. 

> Properties declared without any explicit visibility keyword are defined as public.
> [PHP.net](https://www.php.net/manual/en/language.oop5.visibility.php#language.oop5.visibility-members)

Comment (adevade) [reply src/UrlBuilder.php:29]: But better to be explicit, IMO.

**Pull request merged**